subprojects/logging/src/main/java/org/gradle/internal/logging/console/BuildLogLevelFilterRenderer.java

```diff
@@ -17,12 +17,11 @@
 package org.gradle.internal.logging.console;
 
 import org.gradle.api.logging.LogLevel;
-import org.gradle.internal.logging.events.BatchOutputEventListener;
 import org.gradle.internal.logging.events.LogLevelChangeEvent;
 import org.gradle.internal.logging.events.OutputEvent;
 import org.gradle.internal.logging.events.OutputEventListener;
 
-public class BuildLogLevelFilterRenderer extends BatchOutputEventListener {
+public class BuildLogLevelFilterRenderer implements OutputEventListener {
```

Comment:
Removing it is a change in behavior it turns out. `gradle task -q --console=plain` would show task headers where it didn't used to, so I figured it belongs in a separate change.